packages/react/src/reconciler.ts

```diff
@@ -133,6 +139,12 @@ const reconciler = reactReconciler<
   scheduleDeferredCallback() {},
   cancelDeferredCallback() {},
 
+  // @ts-ignore -- @types/react-reconciler is several versions out of date
+  schedulePassiveEffects(fn:Function) {
+    setTimeout(fn);
+  },
+  cancelPassiveEffects() {},
```

Comment:
I assume this should be implemented as `clearTimeout`?

Reply:
probably. not sure what the function signature is though; `handle: number` i guess...